test/index.html

```diff
@@ -3,12 +3,12 @@
     <head>
         <title>Constructed style sheets test</title>
         <meta charset="UTF-8">
-        <script src="../adoptedStyleSheets.js"></script>
+        <script src="../src/index.js" type="module"></script>
```

Comment:
Better to load from dist or set up a script to run this from localhost otherwise this doesn't work from `file://`

Reply:
Fixed